### PR TITLE
C2B Post for review

### PR DESCRIPTION
**Pull request by Ananya-Joshi**
C2B post in markdown format to review

- Are the images of good quality 
- Do the captions show in markdown 
- Any additional comments about the post.

### CODE REVIEW

**Comment by sgratzl** [comment]
I migrated this PR to https://github.com/cmu-delphi/www-main/pull/257 which is following the blog template

**Comment by Ananya-Joshi** [comment]
Thank you @sgratzl  for your comments! I think all the To-Dos are completed on my end excelt turning the markdown into html.

File: content/blog/2021-03-10_c2b.Rmd
Comment (sgratzl) [inline content/blog/2021-03-10_c2b.Rmd:17]: note that the summary will be cropped after a number of words/characters:

image

File: npm-debug.log
Comment (sgratzl) [inline npm-debug.log:1]: please remove this log file

**Review by sgratzl — CHANGES_REQUESTED** [review]
please ensure the proper format and size of the hero images:
 * large image: 1120x440 in both jpg and webp format - currently: 476x200 in a renamed png format
 * thumbnail: 300x200 in both jpg and webp format - currently: 300x200 in a renamed png format

**Comment by sgratzl** [comment]
I added the webp images in https://github.com/cmu-delphi/www-main/commit/dbdfde054c831b2a1638548adddfa7f9bf206420, please merge the "sgratzl/c2b_blogpost" branch in yours, since I cannot push directly into your forked branch

**Comment by Ananya-Joshi** [comment]
Before merging we will still need approval from the ACHD.

**Review by sgratzl — APPROVED** [review]
good from a technical point of view

File: content/blog/2021-03-10_c2b.Rmd
Comment (bweaver-work) [inline content/blog/2021-03-10_c2b.Rmd:92]: Small nit, but the first two sentences read a little redundant. Maybe "The length of stay is how long someone was in the hospital; it is approximately..."

Comment (Ananya-Joshi) [reply content/blog/2021-03-10_c2b.Rmd:92]: Good catch, how about just the second sentence:


" The length of stay is approximately how many days a person uses a bed in the hospital (± 1 day). This data..."

Comment (bweaver-work) [reply content/blog/2021-03-10_c2b.Rmd:92]: That sounds great to me.

File: content/blog/2021-03-10_c2b.Rmd
Comment (capnrefsmmat) [inline content/blog/2021-03-10_c2b.Rmd:71]:
```suggestion
As the Delphi Group tried to develop solutions to these problems, we consistently ran into data access challenges and bureaucratic limitations to develop solutions at the granularity needed. Carnegie Mellon University did not have a Business Associates Agreement, or BAA, in place with the ACHD. Due to HIPAA laws, that meant we could only operate on public, aggregated and/or redacted data sets. BAAs are commonplace in healthcare, but are complex legal agreements that can take months to execute, even in normal circumstances. Without a BAA in place, we couldn't use detailed COVID testing data classified as Protected Health Information (PHI). Intended projects like setting up mobile testing sites in specific neighborhoods became impossible because public data is aggregated only to the county level. We could not do finer-grained intelligence.
```

File: content/blog/2021-03-10_c2b.Rmd
Comment (capnrefsmmat) [inline content/blog/2021-03-10_c2b.Rmd:88]:
```suggestion
We used public data from Allegheny County about the number of people tested, test positivity rate, and hospitalization rate, broken down by the aforementioned demographic factors. Of these characteristics, we found that age group was the most important.
```

File: content/blog/2021-03-10_c2b.Rmd
Comment (capnrefsmmat) [inline content/blog/2021-03-10_c2b.Rmd:100]:
```suggestion
Monte Carlo simulations involve running a huge number of scenarios based on a set of probabilities. The more scenarios run, the more accurate the model tends to be.  For example, if you gave 1000 people one dart to throw at a dart board, even though each individual throw may not be very good, you’d still be able to get a pretty good idea of where the bull’s eye is after 1000 throws. This is the same principle we applied for Cases2Beds -- after many simulations, we had a good idea of how many beds might be needed in the next two weeks. 
```

File: content/blog/2021-03-10_c2b.Rmd
Comment (capnrefsmmat) [inline content/blog/2021-03-10_c2b.Rmd:104]:
```suggestion
It is relatively straight-forward to port the Python application to VBScript macros.  Those macros, however, aren’t designed to run large simulations and we saw that the time required to generate a model was far, far worse, bordering on unusable.
```

File: data/authors.yaml
Comment (capnrefsmmat) [inline data/authors.yaml:78]: weren't we being picky about writing "Google.org Fellow" at one point?

**Review by capnrefsmmat — COMMENTED** [review]
looking good -- just suggesting some minor copyedits

File: data/authors.yaml
Comment (tildechris) [reply data/authors.yaml:78]: Yeah, it should be Google.org.

**Comment by Ananya-Joshi** [comment]
Blog post is ready.

**Comment by tildechris** [comment]
@Ananya-Joshi one final suggestion: if you have a Date axis and it literally means calendar date (like in your estimation plots), you can omit the x-axis label.  Roni had the suggestion, however, that we can be more precise in x-axis by saying "reported date" (from COVIDcast or similar) or whatever the date label is from the source (e.g. collection date)

File: content/blog/2021-03-10_c2b.Rmd
Comment (sgratzl) [inline content/blog/2021-03-10_c2b.Rmd:65]: I thought I moved the files and updated the links in my update PR...

Comment (Ananya-Joshi) [reply content/blog/2021-03-10_c2b.Rmd:65]: Hm I wonder what happened here, I'll undo my last commit

Comment (tildechris) [reply content/blog/2021-03-10_c2b.Rmd]: Merged. Please take a look.

**Pull request merged**
